Documentation/BuildInstructions.md

```diff
@@ -169,3 +169,23 @@ Later on, when you `git pull` to get the latest changes, there's (usually) no ne
 
 #### Ports
 To add a package from the ports collection to Serenity, for example curl, go into `Ports/curl/` and run **./package.sh**. The sourcecode for the package will be downloaded and the package will be built. After that, run **make image** from the `Build/` directory to update the disk image. The next time you start Serenity with **make run**, `curl` will be available.
+
+#### Keymap
+
+Create a file with the exact name `sync-local.sh` in the project root (the same directory as `.clang-format`), with content like this:
+
+```
+#!/bin/sh
+
+set -e
+
+cat << 'EOF' >> mnt/etc/SystemServer.ini
+
+[keymap]
+Executable=/bin/keymap
+Arguments=de
+User=anon
+EOF
+```
+
+This will configure your keymap to German (`de`) instead of English. See [`Base/res/keymaps/`](../Base/res/keymaps/) for a full list.
```

Comment:
```suggestion
This will configure your keymap to German (`de`) instead of US English. See [`Base/res/keymaps/`](../Base/res/keymaps/) for a full list.
```

Reply:
we should also rename `en.json` to `us.json` or `en.json` to `enus.json` and `gb.json` to `engb.json` (for consistency with `ptpt` and `ptbr`), no?

Reply:
Sounds good, yeah. I'd prefer `en-us`, `en-gb`, `pt-...` though.

Reply:
agreed

Reply:
Okay, this is *definitely* out of scope for this PR. I mainly wanted to get the German layout complete and all tools working properly.

(Oh and: language-country codes are usually written like `pt_BR`, aren't they?)

Reply:
@BenWiederhake: this is the full output of `locate keymaps` on my computer: https://gist.github.com/emanuele6/d89f513943fa26d67f0a8d1f6c6c294c

`pt-br` seems to be how is usually written. `pt_BR` looks like the name format of `locale` files.